cmd/lncli/commands.go

```diff
@@ -111,13 +111,45 @@ var SendCoinsCommand = cli.Command{
 }
 
 func sendCoins(ctx *cli.Context) error {
+	var addr string = ""
+	var amt int64 = 0
+	var args cli.Args = ctx.Args()
+
+	if ctx.NArg() == 0 && ctx.NumFlags() == 0 {
+		cli.ShowCommandHelp(ctx, "sendcoins")
+		return nil
+	}
+
+	if ctx.IsSet("addr") {
+		addr = ctx.String("addr")
+	} else if args.Present() {
+		addr = args.First()
+		args = args.Tail()
+	} else {
+		return fmt.Errorf("Address argument missing")
+	}
+
+	var err error
```

Comment:
This error declaration should be hoisted up to the top where all the other variables are initially declared.